### PR TITLE
New: Deploy a randomized NFT drop without writing any code

### PR DESCRIPTION
**Pull request by warengonzaga**
Hey @nachoiacovino kindly review the updated version of this guide. Thanks!

### CODE REVIEW

File: guides/randomized-nft-drop/index.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+Let’s assume you’ve already a MetaMask wallet, if not please follow our [create a MetaMask wallet tutorial](https://portal.thirdweb.com/guides/create-a-metamask-wallet). We’re going to use Polygon Testnet Mumbai for this guide instead of using Mainnets (Ethereum, Polygon, Fantom, Avalanche, etc...). We’ll be needing some test balance for the Mumbai Testnet, please follow our tutorial on [how to get Matic on Polygon Mumbai Testnets](https://portal.thirdweb.com/guides/get-matic-on-polygon-mumbai-testnet-faucet). Lastly, the image assets to use! Feel free to use the same images I use for this guide, you can [download the image assets here](./random_eyes_assets.zip).
+
+Now, let’s head over to the [thirdweb dashboard](https://thirdweb.com/dashboard) and **create a new contract** and select **Mumbai**.
+
+![Create a new nft drop contract](./create-a-new-nft-drop-contract.png)
+
+After that, you’ll be asked on the next screen *What do you want to build?* Just select **Release a drop** then **NFT drop**.
+
+![Choose 'Release a drop'](./choose-a-release-drop.png)
```
Comment (jarrodwatts) [inline guides/randomized-nft-drop/index.mdx:34]: This image is not how it looks anymore, the UI is different

Comment (jarrodwatts) [reply guides/randomized-nft-drop/index.mdx:34]: - [ ] Update the image and text above this image to reflect what the new UI says + looks like

File: guides/randomized-nft-drop/index.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+Here’s my **NFT drop contract** information, fill it up with your own information. For this guide, I’ll keep the percentage **as is** but you can put whatever you want value. The ideal for me would be 5% but still, it’s up to you. After that, **click deploy** and **confirm the transaction**.
```
Comment (jarrodwatts) [inline guides/randomized-nft-drop/index.mdx:36]: - [ ] Re-phrase this to "Next you'll need to fill out the details of your NFT Drop. For this guide, Specify a name, symbol, description, and configure the royalties you want to earn on secondary sales of your NFTs. After that, **click deploy** and **confirm the transaction**."

File: guides/randomized-nft-drop/index.mdx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+![Fill up the contract form](./fill-up-the-contract-form.png)
+
+Once the transaction has been confirmed you will see the dashboard of your NFT drop contract and see something like this below.
```
Comment (jarrodwatts) [inline guides/randomized-nft-drop/index.mdx:40]: - [ ] Re-phrase this to "Once the transaction has been confirmed, you can see and configure your NFT Drop contract within the dashboard."

File: guides/randomized-nft-drop/index.mdx (hunk 1 above)
Comment (warengonzaga) [reply guides/randomized-nft-drop/index.mdx:34]: thanks @jarrodwatts, I would probably do a quick run down to the first part of the tutorial to match the latest UX for creation of contract.

**Comment by warengonzaga** [comment]
Hey @jarrodwatts I update the whole guide from bottom to top with latest UI/UX.

**Pull request merged**